examples/variables-1.md

```diff
@@ -20,20 +20,26 @@ filter ct > 200
 take 20
 ```
 
-```sql
-SELECT TOP 20
-    title,
-    country,
-    AVG(salary) AS average_salary,
-    SUM(salary) AS sum_salary,
-    AVG(salary + payroll_tax) AS average_gross_salary,
-    SUM(salary + payroll_tax) AS sum_gross_salary,
-    AVG(salary + payroll_tax + benefits_cost) AS average_gross_cost,
-    SUM(salary + payroll_tax + benefits_cost) AS sum_gross_cost,
-    COUNT(*) as count
-FROM employees
-WHERE salary + payroll_tax + benefits_cost > 0 AND country = 'USA'
-GROUP BY title, country
-ORDER BY sum_gross_cost
-HAVING count > 200
-```
+```elm
+SELECT
+  TOP (20) title,
+  country,
+  SUM(salary + payroll_tax + benefits_cost) AS sum_gross_cost,
+  COUNT(*) AS ct,
+  AVG(salary),
+  SUM(salary),
+  AVG(salary + payroll_tax),
+  SUM(salary + payroll_tax),
+  AVG(salary + payroll_tax + benefits_cost)
+FROM
+  employees
+WHERE
+  country = 'USA'
+  and salary + payroll_tax + benefits_cost > 0
+GROUP BY
+  title,
+  country
+HAVING
+  ct > 200
+ORDER BY
+  sum_gross_cost```
```

Comment:
The lack of a newline causes it to have the closing backticks in the visible markdown:
<img width="244" alt="image" src="https://user-images.githubusercontent.com/5635139/159550147-8bdf60fd-9f27-4e4f-97a5-244e22df9cae.png">

Reply:
Whoops